res/controllers/Pioneer-DDJ-200-scripts.js

```diff
@@ -1,41 +1,133 @@
 var DDJ200 = {
-    jogDisabled: new Array(false, false, false, false, false)
+    fourDeckMode: false,
+    vDeckNo: [0, 1, 2],
+    vDeck: {},
+    shiftPressed: {left: false, right: false},
+    jogCounter: 0
 };
 
-DDJ200.init = function() {};
+DDJ200.init = function() {
+    for (var i = 1; i <= 4; i++) {
+                                  
+       // create associative arrays for 4 virtual decks
+        this.vDeck[i] = {
+        syncEnabled: false,
+        volMSB: 0,
+        rateMSB: 0,
+        jogDisabled: false,
```

Comment:
It's a bit confusing that this is inverted.

How about this?
```suggestion
        syncEnabled: false,
        volMSB: 0,
        rateMSB: 0,
        jogEnabled: true,
```